src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -28,7 +28,9 @@ public int CreateEmptyProject()
         {
             var thisAssembly = typeof(Program).GetTypeInfo().Assembly;
             var resources = from resourceName in thisAssembly.GetManifestResourceNames()
-                            where resourceName.ToLowerInvariant().EndsWith(".cs") || resourceName.ToLowerInvariant().EndsWith(".json")
+                            where resourceName.ToLowerInvariant().EndsWith(".cs")
+                            || resourceName.ToLowerInvariant().EndsWith(".json")
+                            || resourceName.ToLowerInvariant().EndsWith(".config")
```

Comment:
I like this. Will make for 0 code changes when adding/removing things to be dropped.